pyqt-apps/siriushla/li_rf_llrf/main.py

```diff
@@ -35,64 +38,170 @@ def __new__(cls, value, label, pvname, nickname):
         return self
 
 
-class MainWindow(SiriusMainWindow):
+class MainWindow(QWidget):
     """."""
 
     def __init__(self, parent=None, prefix=_VACA_PREFIX):
         """."""
         super().__init__(parent=parent)
-        self.prefix = prefix
+        self.prefix = prefix + ('-' if prefix else '')
+        self.display_format = DisplayFormat
+        self.main_dev = 'LA-RF:LLRF:'
         self.setObjectName('LIApp')
         self.setWindowTitle('LI LLRF')
-        self.setWindowIcon(_qta.icon(
-            'mdi.waves', color=_util.get_appropriate_color('LI')))
-
+        self.image_container = QLabel()
+        self.pixmap = QPixmap(_os.path.join(
+            _os.path.abspath(_os.path.dirname(__file__)), "llrf.png"))
+        self.relative_widgets = []
         self._setupui()
 
+    def resizeEvent(self, event):
+        """Signal the resize event to the relative Widgets"""
+        for relative_item in self.relative_widgets:
+            relative_item.relativeResize()
+
+    def buildPvName(self, pv_name, device, pvPrefix='', pvSufix=''):
+        """Build the pv name"""
+        return (self.prefix + self.main_dev +
+            device + ":" + pvPrefix + pv_name + pvSufix)
+
+    def imageViewer(self):
+        """Build the image"""
+        self.image_container.setPixmap(self.pixmap)
+        self.image_container.setScaledContents(True)
+        self.image_container.setSizePolicy(
+            QSizePolicy.Ignored, QSizePolicy.Ignored)
+        self.image_container.setMinimumSize(950, 0)
+        return self.image_container
+
+    def formatLabel(self, pv_name='', pv_type='Power'):
+        """Get widget type"""
+        widget = SiriusLabel(init_channel=pv_name)
+        widget.precisionFromPV = False
+        widget.precision = 2
+        if pv_type == 'Power':
+            widget._display_format_type = self.display_format.Exponential
```

Comment:
you can use the property instead of the private variable: `widget.displayFormat = ...`